itests/self_sent_txn_test.go

```diff
@@ -102,3 +102,23 @@ func TestSelfSentTxnV14(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, exitcode.Ok, mLookup.Receipt.ExitCode)
 }
+
+func TestStateLookupRobustAddress(t *testing.T) {
+	ctx := context.Background()
+	kit.QuietMiningLogs()
+
+	client15, miner, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.GenesisNetworkVersion(network.Version15))
```

Comment:
```suggestion
	client, miner, ens := kit.EnsembleMinimal(t, kit.MockProofs(), kit.GenesisNetworkVersion(network.Version15))
```

Rename this to just `client`? `client15` was relevant to the other test in this file (it was relevant to network v15).